js/src/common/utils/proxifyCompat.ts

```diff
@@ -1,10 +1,12 @@
 export default (compat: { [key: string]: any }, namespace: string) => {
   // regex to replace common/ and NAMESPACE/ for core & core extensions
+  // and remove .js, .ts and .tsx extensions
   // e.g. admin/utils/extract --> utils/extract
   // e.g. tags/common/utils/sortTags --> tags/utils/sortTags
-  const regex = new RegExp(`(\\w+\\/)?(${namespace}|common)\\/`);
+  const regex = new RegExp(`^(.+\/)?(${namespace}|common)\/`);
```

Comment:
Why change the part of the regex before the namespace?

Reply:
i don't remember

Reply:
I think we should revert that. If we don't remember what it does, and the old code works fine, there's no need to change it.